spec/exchanges/bitconnect/integration/market_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+RSpec.describe 'Bitconnect integration specs' do
+  client = Cryptoexchange::Client.new
+
+  it 'fetch pairs' do
+    pairs = client.pairs('bitconnect')
+    expect(pairs).not_to be_empty
+
+    pair = pairs.first
+    expect(pair.base).to_not be nil
+    expect(pair.target).to_not be nil
+    expect(pair.market).to eq 'bitconnect'
+  end
+
+  it 'fetch ticker' do
+    pair = Cryptoexchange::Models::MarketPair.new(base: 'BTC', target: 'BCC', market: 'bitconnect')
```

Comment:
Specs requires fixing too after fixing https://github.com/coingecko/cryptoexchange/pull/219/files#r152625433

Reply:
Corrected to `base: BCC`, `target: BTC`